pages/governance.md

```diff
@@ -0,0 +1,235 @@
+possible that Members will have conflicts of interest. Such conflicts of
+interest include, but are not limited to:
+
+* Financial interests, such as investments, employment or contracting work,
+  outside of JuMP that may influence their work on JuMP.
+* Access to proprietary information of their employer that could potentially
+  leak into their work with JuMP.
+
+All members of the Steering Council shall disclose to the Steering Council any
+conflict of interest they may have. Members with a conflict of interest in a
+particular issue may participate in Steering Council discussions on that issue,
+but must recuse themselves from authorizing or approving any expenditures.
+
+## Decision Making Process
+
+This section outlines how fiscal and non-fiscal decisions are made in the JuMP
+project.
+
+### Fiscal Decisions
+
+All fiscal decisions are made by the Steering Council to ensure any funds are
+spent in a manner that furthers the mission of JuMP. Fiscal decisions require
+majority approval by acting Steering Council members.
+
+The Steering Council can gain addition decision-making power if the core
+contributors decide to delegate, e.g., choosing a chair for a JuMP-dev workshop.
+
+Community members proposing decisions with a fiscal aspect should contact the
+Steering Council directly, or table their proposal as an agenda item for
+discussion on a Monthly developer call (see below).
+
+### Non-fiscal Decisions
+
+All non-fiscal decisions are made via consensus of the core contributors and
+relevant repository maintainers. Emeritus core contributors are not considered
+to be core contributors for this purpose.
+
+Code-related decisions, such when a pull request is ready to be accepted and
+merged, should be discussed via the relevant Github issues and pull requests. If
+consensus cannot be achieved, the community member proposing the change may be
+invited by a core contributor to present their proposal at a Monthly developer
+call for further discussion and community input.
+
+Non-code-related technical decisions, such long-term strategic planning for
+JuMP, should either be discussed in a Github issue, or tabled as an agenda item
+and discussed on a Monthly developer call.
+
+If consensus on a non-fiscal decision cannot be achieved, the final decision
+will be made by the BDFL.
+
+## Github Permissions
+
+For security, all core contributors and repository maintainers must have
+two-factor authentication enabled through their Github account.
+
+### Core Contributors
+
+Core contributors are added as [Owners](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization#permission-levels-for-an-organization)
+of the [JuMP-dev Github organization] and have [Admin][permissions] permission
+to every repository.
+
+### Emeritus Core Contributors
+
+Unless directed, emeritus core contributors retain the same commit rights as
```

Comment:
"Emeritus core contributors retain the same commit rights as core contributors, unless they choose to surrender them."